libnavigation-base/src/test/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectMapperTest.kt

```diff
@@ -149,6 +153,7 @@ class RoadObjectMapperTest {
             LENGTH,
             location,
             SDKRoadObjectProvider.MAPBOX,
+            false,
```

Comment:
maybe also add ?
```kotlin
assertEquals(expected. isUrban, roadObject.isUrban)
```

Reply:
tests are `@Ignored`, it's only to make project compiled